README.md

```diff
@@ -131,3 +132,63 @@ _**NOTE:** Published versions can be installed from the [Chrome Web Store](https
 ### MIT License
 
 * [Font-Awesome-SVG-PNG v1.1.5](https://github.com/encharm/Font-Awesome-SVG-PNG)
+
+## Publishing to the Chrome Web Store
+
+1. Prepare Code
+   1. Ensure that all of the code, that should be published, has been merged into the master branch.
+   2. Ensure that the code in master is working as expected
+      1. Run tests: `npm test`
+      2. Lint: `grunt lint`
+      3. Manual test build
+         1. Create a developer build: "grunt buildDev"
+         2. Load unpacked extension into Chrome
+            1. In Chrome go to [chrome://extensions](chrome://extensions)
+            2. Ensure that "Developer mode" is enabled
+            3. Click "Load unpacked"
+            4. From the File Dialog, navigate to the "gpii-chrome-extension" repo and select the "build" directory.
+         3. Test all of the preferences and ensure that they apply to web page content correctly.
+            1. Refresh any Browser Tabs/Windows that were open prior to installing the extension.
+            2. The preferences should be tested individually and in combinations to ensure that interactions between the
+               preferences are working properly. For example (Text-To-Speech and Syllabification).
+            3. Multiple web pages should be tested. However, not all preferences will work with all sites. For example,
+               YouTube captions will only work with YouTube videos embedded using the iframe API and providing captions
```

Comment:
Would be nice to add [the exemplary youtube video](https://build.fluidproject.org/infusion/examples/framework/preferences/captionsPreference/) you created here to provide a testable case for youtube captions.